packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -20,7 +20,11 @@ import 'view.dart';
 export 'package:flutter/gestures.dart' show HitTestResult;
 
 /// The glue between the render tree and the Flutter engine.
-abstract class RendererBinding extends BindingBase implements SchedulerBinding, ServicesBinding, HitTestable {
+abstract class RendererBinding extends BindingBase with SchedulerBinding, ServicesBinding, HitTestable {
```

Comment:
not sure how to add Peter to this comment, he should be able to confirm this, but from what I can see DFE needs concrete instance methods(this does match what the spec says if I read it right: https://github.com/gbracha/lessRestrictedMixins/blob/master/proposal.md#16151-method-lookup)